shared/js/background/events.es6.js

```diff
@@ -384,7 +384,8 @@ function getArgumentsObject (tabId, sender, documentUrl) {
     }
     // Special case for iframes that are blank we check if it's also enabled
     if (sender.url === 'about:blank') {
-        site.brokenFeatures = site.brokenFeatures.concat(utils.getBrokenFeaturesAboutBlank(tab.url))
+        const aboutBlankEnabled = utils.getEnabledFeaturesAboutBlank(tab.url)
+        site.enabledFeatures = site.enabledFeatures.filter(feature => aboutBlankEnabled.includes(feature))
```

Comment:
Shouldn't this be:
```suggestion
        site.enabledFeatures = site.enabledFeatures.filter(feature => !aboutBlankEnabled.includes(feature))
```

Reply:
We want intersection of "features enabled in general" and "features enabled in about:blank iframes" right? If so this code looks good to me:

```js
const generalEnabled = ['a', 'b', 'c'];
const blankEnabled = ['b', 'c', 'd'];

generalEnabled.filter(feature => blankEnabled.includes(feature)) // results in ['b', 'c'] as expected
```

Reply:
Agreed thanks :)